docs/src/validator/get-started/setup-a-validator.md

```diff
@@ -0,0 +1,422 @@
+
+```
+sudo apt update
+sudo apt install
+```
+
+## Sol User
+
+Create a new Ubuntu user, named `sol`, for running the validator:
+
+```
+sudo adduser sol
+```
+
+It is a best practice to always run your validator as a non-root user, like the `sol` user we just created.
+
+## Hard Drive Setup
+
+On your Ubuntu computer make sure that you have at least `2TB` of disk space mounted. You can check disk space using the `df` command:
+
+```
+df -h
+```
+
+> If you have a drive that is not mounted/formatted, you will have to set up the partition and mount the drive.
+
+To see the hard disk devices that you have available, use the list block devices command:
+
+```
+lsblk -f
+```
+
+You may see some devices in the list that have a name but do not have a UUID. Any device without a UUID is unformatted.
+
+### Drive Formatting: Ledger
+
+Assuming you have an nvme drive that is not formatted, you will have to format the drive and then mount it.
+
+For example, if your computer has a device located at `/dev/nvme0n1`, then you can format the drive with the command:
+
+```
+sudo mkfs -t ext4 /dev/nvme0n1
+```
+
+For your computer, the device name and location may be different.
+
+Next, check that you now have a UUID for that device:
+
+```
+lsblk -f
+```
+
+In the fourth column, next to your device name, you should see a string of letters and numbers that look like this: `6abd1aa5-8422-4b18-8058-11f821fd3967`. That is the UUID for the device.
+
+### Mounting Your Drive: Ledger
+
+So far we have created a formatted drive, but you do not have access to it until you mount it. Make a directory for mounting your drive:
+
+```
+sudo mkdir -p /mnt/ledger
+```
+
+Next, change the ownership of the directory to your `sol` user:
+
+```
+sudo chown -R sol:sol /mnt/ledger
+```
+
+Now you can mount the drive:
+
+```
+sudo mount /dev/nvme0n1 /mnt/ledger
+```
+
+### Formatting And Mounting Drive: AccountsDB
+
+You will also want to mount the accounts db on a separate hard drive. The process will be similar to the ledger example above.
+
+Assuming you have device at `/dev/nvme1n1`, format the device and verify it exists:
+
+```
+sudo mkfs -t ext4 /dev/nvme1n1
+```
+
+Then verify the UUID for the device exists:
+
+```
+lsblk -f
+```
+
+Create a directory for mounting:
+
+```
+sudo mkdir -p /mnt/accounts
+```
+
+Change the ownership of that directory:
+
+```
+sudo chown -R sol:sol /mnt/accounts
+```
+
+And lastly, mount the drive:
+
+```
+sudo mount /dev/nvme1n1 /mnt/accounts
+```
+
+## System Tuning
+
+In order for your validator to run properly, you will need to tune the system. (If you skip this step, you will likely get an out of memory error).
+
+> You can find more advanced information about [manual tuning options](../../running-validator/validator-start#system-tuning) here.
+
+To get started, the automated `solana-sys-tuner` will work well for us. Run the following:
+
+```
+sudo $(command -v solana-sys-tuner) --user sol > sys-tuner.log 2>&1
```

Comment:
Is it still reccomended to use systuner? It might be worth talking to mvines or trent about this piece.

Reply:
Yeah, good point. I'll take that out.